### ISSUE TITLE
Is Sigmoid not supported yet?

### ISSUE THREAD

**Issue by aknirala** [Verified-Intelligence/auto_LiRPA#83]
When I use Sigmoid, in a neural network, the library does not return BoundedModule. Is this because, Sigmoid is not supported yet?

For reference here is my code:

```
import torch
import torch.nn as nn
from auto_LiRPA import BoundedModule
import torch
import torch.nn as nn

class DoubleLinearSigmoidNetwork(nn.Module):
    def __init__(self):
        super(DoubleLinearSigmoidNetwork, self).__init__()
        # First linear layer with weights set to identity and bias set to zero
        self.linear1 = nn.Linear(10, 10)
        self.sigmoid = nn.Sigmoid()
        self.linear2 = nn.Linear(10, 10)

    def forward(self, x):
        x = self.linear1(x)
        x = self.sigmoid(x)
        x = self.linear2(x)
        return x

network = DoubleLinearSigmoidNetwork()
input_data = torch.tensor([[1.0, 2.0, -1.0, 0.0, 3.0, 4.0, 5.0, -2.0, 6.0, 7.0]])  # Example input of size 10
output = network(input_data)
print(output)

dummy_input = torch.randn(2, 10)
model_b = BoundedModule(network, dummy_input, bound_opts={'relu':"same-slope"}, device='cpu')
model_b(input_data)
```

**Comment by shizhouxing** [comment]
Sigmoid is supported. What do you mean by "library does not return BoundedModule"?
